lib/cfg-grammar.y

```diff
@@ -895,6 +896,8 @@ template_content
         : { last_template = log_template_new(configuration, NULL); } template_content_inner	{ $$ = last_template; }
         ;
 
+template_content_list : template_content template_content_list { $$ = g_list_prepend($2, $1); } | { $$ = NULL; }
+
```

Comment:
Sorry, it was just pseudocode, I forgot to add `;`:

```
template_content_list
	:  template_content template_content_list { $$ = g_list_prepend($2, $1); }
	| { $$ = NULL; }
	;
```

Reply:
Thanks, fixed! :)